main.py

```diff
@@ -1,7 +1,7 @@
 from mininet.clean import Cleanup
 from mininet.net import Mininet
 from mininet.cli import CLI
-
+from topologies.clos import CLOS
```

Comment:
You should use conditional import to make the code concise and flexible
Eg. 
if args.topo = 'clos':
  from topologies.clos import CLOS as CustomTopo
else:
  from topologies.ocs import OCS as CustomTopo